### ISSUE TITLE
Light Card Tap Icon

### ISSUE THREAD

**Issue by padermon** [UI-Lovelace-Minimalist/UI#79]
Hey,

Is it possible to create a tap action just for the icon of the light card?
It would be nice if i tap on the icon the details pages opens and if i tap on the card it toggles the light.

Do you know if it is possible?

Thanks in advance!

**Comment by basbruss** [comment]
As for now it's not possible to have different actions for the icon and the label & name. 
Or we need to build a card which consists of two items next to each other which separates the icon from the label and name.
You could define a double_tap_action or a hold_action (already supported) to show more-info. As they are part of the same button.
Just add in your lovelace-file
```
- type: 'custom:button-card'
  template: card_light
  double_tap_action:
    action: more-info
```
But I have already implemented something similar to this in a custom light_slider card I have made.
<img width="242" alt="Screen Shot 2021-12-14 at 16 07 05" src="https://user-images.githubusercontent.com/68892092/146024440-034292ac-3611-4b08-9414-e4d6bdf3c859.png">
Here does the button on the bottom-right serve as a more-info button.

I can prepare a PR to give additional functionality to the light-buttons.

**Comment by padermon** [comment]
Your custom card is awesome.
Can you make a PR that I can use your card?
Thanks!

**Comment by basbruss** [comment]
I have opened a PR with custom colored cards and the basic light card with the icon as more-info button as you requested haha :) PR https://github.com/UI-Lovelace-Minimalist/UI/pull/80

**Comment by CM000n** [comment]
Closed in v1.0.0-beta.8 🏳️‍🌈
